starter-code/index.html

```diff
@@ -2,30 +2,45 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+  <div class="background">
```

Comment:
Inspect your background div and you'll see everything is not fitting inside it. 

Maybe you could put everything outside the background div. Actually, you want the background div to act as body tag is doing; then you could just define backround to body.

You could also put all the stuff (images, titles, text, buttons) in a big container and try to center it.